<?xml version="1.0" encoding="UTF-8"?>
<testng-results retried="1" ignored="0" total="4" passed="2" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2022-06-03T15:40:03 EEST" name="TestNGXMLTest Test Suite" finished-at="2022-06-03T15:41:09 EEST" duration-ms="65951">
    <groups>
    </groups>
    <test started-at="2022-06-03T15:40:03 EEST" name="TestNGXMLTest Test" finished-at="2022-06-03T15:41:09 EEST" duration-ms="65951">
      <class name="org.example.test.AddToWishlistTest">
        <test-method is-config="true" signature="setUp(org.testng.ITestContext)[pri:0, instance:org.example.test.AddToWishlistTest@607b2792]" started-at="2022-06-03T15:40:59 EEST" name="setUp" finished-at="2022-06-03T15:41:05 EEST" duration-ms="5983" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[org.testng.TestRunner@3ba8ef4e]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="verifyNotAbleToAddToWishlistWithoutLoginTest()[pri:0, instance:org.example.test.AddToWishlistTest@607b2792]" started-at="2022-06-03T15:41:05 EEST" name="verifyNotAbleToAddToWishlistWithoutLoginTest" description="Verify that customer is not able to add item to wishlist without login in" finished-at="2022-06-03T15:41:09 EEST" duration-ms="3839" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyNotAbleToAddToWishlistWithoutLoginTest -->
        <test-method is-config="true" signature="tearDown(org.testng.ITestResult)[pri:0, instance:org.example.test.AddToWishlistTest@607b2792]" started-at="2022-06-03T15:41:09 EEST" name="tearDown" finished-at="2022-06-03T15:41:09 EEST" duration-ms="118" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=verifyNotAbleToAddToWishlistWithoutLoginTest status=SUCCESS method=AddToWishlistTest.verifyNotAbleToAddToWishlistWithoutLoginTest()[pri:0, instance:org.example.test.AddToWishlistTest@607b2792] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- org.example.test.AddToWishlistTest -->
      <class name="org.example.test.HomePageTest">
        <test-method is-config="true" signature="setUp(org.testng.ITestContext)[pri:0, instance:org.example.test.HomePageTest@28c88600]" started-at="2022-06-03T15:40:03 EEST" name="setUp" finished-at="2022-06-03T15:40:15 EEST" duration-ms="12301" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[org.testng.TestRunner@3ba8ef4e]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="homePageTest()[pri:0, instance:org.example.test.HomePageTest@28c88600]" started-at="2022-06-03T15:40:16 EEST" name="homePageTest" description="Verify that customer is on home page" finished-at="2022-06-03T15:40:16 EEST" duration-ms="300" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- homePageTest -->
        <test-method is-config="true" signature="tearDown(org.testng.ITestResult)[pri:0, instance:org.example.test.HomePageTest@28c88600]" started-at="2022-06-03T15:40:16 EEST" name="tearDown" finished-at="2022-06-03T15:40:16 EEST" duration-ms="142" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=homePageTest status=SUCCESS method=HomePageTest.homePageTest()[pri:0, instance:org.example.test.HomePageTest@28c88600] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- org.example.test.HomePageTest -->
      <class name="org.example.test.RegistrationTest">
        <test-method is-config="true" signature="setUp(org.testng.ITestContext)[pri:0, instance:org.example.test.RegistrationTest@5f8890c2]" started-at="2022-06-03T15:40:16 EEST" name="setUp" finished-at="2022-06-03T15:40:24 EEST" duration-ms="8130" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[org.testng.TestRunner@3ba8ef4e]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method retried="true" signature="verifySuccessfulRegistrationTest()[pri:0, instance:org.example.test.RegistrationTest@5f8890c2]" started-at="2022-06-03T15:40:24 EEST" name="verifySuccessfulRegistrationTest" description="Verify customer is able to login in" finished-at="2022-06-03T15:40:38 EEST" duration-ms="13611" status="SKIP">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[null]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: null
at org.testng.Assert.fail(Assert.java:97)
at org.testng.Assert.fail(Assert.java:102)
at org.example.test.RegistrationTest.verifySuccessfulRegistrationTest(RegistrationTest.java:58)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:134)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:597)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:816)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:766)
at org.testng.TestRunner.run(TestRunner.java:587)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1109)
at org.testng.TestNG.runSuites(TestNG.java:1039)
at org.testng.TestNG.run(TestNG.java:1007)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:284)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:119)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:428)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:162)
at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:562)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:548)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifySuccessfulRegistrationTest -->
        <test-method is-config="true" signature="tearDown(org.testng.ITestResult)[pri:0, instance:org.example.test.RegistrationTest@5f8890c2]" started-at="2022-06-03T15:40:38 EEST" name="tearDown" finished-at="2022-06-03T15:40:38 EEST" duration-ms="88" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=verifySuccessfulRegistrationTest status=SKIP method=RegistrationTest.verifySuccessfulRegistrationTest()[pri:0, instance:org.example.test.RegistrationTest@5f8890c2] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="setUp(org.testng.ITestContext)[pri:0, instance:org.example.test.RegistrationTest@5f8890c2]" started-at="2022-06-03T15:40:38 EEST" name="setUp" finished-at="2022-06-03T15:40:44 EEST" duration-ms="6583" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[org.testng.TestRunner@3ba8ef4e]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="verifySuccessfulRegistrationTest()[pri:0, instance:org.example.test.RegistrationTest@5f8890c2]" started-at="2022-06-03T15:40:44 EEST" name="verifySuccessfulRegistrationTest" description="Verify customer is able to login in" finished-at="2022-06-03T15:40:58 EEST" duration-ms="13777" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[null]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: null
at org.testng.Assert.fail(Assert.java:97)
at org.testng.Assert.fail(Assert.java:102)
at org.example.test.RegistrationTest.verifySuccessfulRegistrationTest(RegistrationTest.java:58)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:134)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:597)
at org.testng.internal.TestInvoker.retryFailed(TestInvoker.java:213)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:58)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:816)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:766)
at org.testng.TestRunner.run(TestRunner.java:587)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1109)
at org.testng.TestNG.runSuites(TestNG.java:1039)
at org.testng.TestNG.run(TestNG.java:1007)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:284)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:119)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:428)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:162)
at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:562)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:548)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifySuccessfulRegistrationTest -->
        <test-method is-config="true" signature="tearDown(org.testng.ITestResult)[pri:0, instance:org.example.test.RegistrationTest@5f8890c2]" started-at="2022-06-03T15:40:59 EEST" name="tearDown" finished-at="2022-06-03T15:40:59 EEST" duration-ms="136" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=verifySuccessfulRegistrationTest status=FAILURE method=RegistrationTest.verifySuccessfulRegistrationTest()[pri:0, instance:org.example.test.RegistrationTest@5f8890c2] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- org.example.test.RegistrationTest -->
    </test> <!-- TestNGXMLTest Test -->
  </suite> <!-- TestNGXMLTest Test Suite -->
</testng-results>
